backend/src/main/java/ca/gov/dtsstn/cdcp/api/web/validation/AlertTypeCodeValidator.java

```diff
@@ -19,7 +20,7 @@ public AlertTypeCodeValidator(AlertTypeService alertTypeService) {
 
 	@Override
 	public boolean isValid(String value, ConstraintValidatorContext context) {
-		if (value == null) { return true; }
+		if (value == null || StringUtils.isBlank(value)) { return true; }
```

Comment:
Since `StringUtils.isBlank(..)` handles the `null` check:

```suggestion
		if (StringUtils.isBlank(value)) { return true; }
```

Reply:
done